code/modules/language/language_holder.dm

```diff
@@ -218,19 +218,24 @@ Key procs
 
 /// Copies all languages from the supplied atom/language holder. Source should be overridden when you
 /// do not want the language overwritten by later atom updates or want to avoid blocked languages.
-/datum/language_holder/proc/copy_languages(var/datum/language_holder/from_holder, source_override)
+/datum/language_holder/proc/copy_languages(var/datum/language_holder/from_holder, source_override=FALSE, spoken=TRUE, understood=TRUE, blocked=TRUE)
 	if(source_override)	//No blocked languages here, for now only used by ling absorb.
```

Comment:
this is the same as the blocked argument

Reply:
these two arguments serve different purposes. At least I changed the logic, so you can recognise how it's different.